test/unit-test/test_lib/test_analysis/test_user.py

```diff
@@ -1,34 +1,135 @@
 import unittest
 from lib.analysis import user
+import lib.config as config
+from lib.in_out import reader, saver
+import lib.nickTracker as nickTracker
+from mock import patch
+import networkx as nx
 import lib.util as util
 import os
+import StringIO
+import sys
+import filecmp
 
 current_directory = os.path.dirname(os.path.realpath(__file__))
+state0 = 0
+state1 = 0
+def build_graphs_util(nick_sender, nick_receiver, time, year, month, day, day_graph, aggr_graph):
```

Comment:
Considering that the `build_graphs_util()` function is used only by the functions of UserTest, it is better to move this function to inside the UserTest class and make it private with two underscores prefix. Please see the notation for private functions [here](http://radek.io/2011/07/21/private-protected-and-public-in-python/) and [here](https://stackoverflow.com/questions/7609508/is-it-a-convention-to-prefix-private-classes-with-underscores).